install/installcore.php

```diff
@@ -13,7 +13,7 @@
 const CURRENT_VERSION = '1.1';
 const CURRENT_LANG_VERSION = '1.1';
 const DB_SCRIPT_VERSION = '1-1';
-const REQUIRED_PHP_VERSION = '5.3.3';
+const REQUIRED_PHP_VERSION = '5.5.0';
```

Comment:
If this is the case you should also update the README.

Reply:
Good catch, done!